regen-network/regen-web
# feat: optional description in TxModal

## Description

Closes: regen-network/regen-registry#1094

- Adds an optional description to the TxModal



---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...


#### Setting up backport PRs

After merging your PR to `dev`, set up backports by doing the following:

1. If your branch does not have merge commits, add the following comment to
   your PR:

- If PR is a new feature: `@Mergifyio backport redwood hambach`.
- If PR is a bug fix: `@Mergifyio backport redwood hambach master`.

2. If your branch does have merge commits:

   a. Pull latest `dev`, `hambach` and `redwood` branches

   b. Create new branches for backports and merge `dev` (replace `<PR#>` with your PR #)

   ```
   git checkout -b hambach-backport-<PR#> hambach
   git merge dev
   git push origin hambach-backport-<PR#>
   git checkout -b redwood-backport-<PR#> redwood
   git merge dev
   git push origin redwood-backport-<PR#>`
   ```

   c. Open new PRs in regen-web targeting `hambach` and `redwood`, respectively.

   d. If PR is a bug fix repeat the same process for `master` branch.

### How to test

1. https://deploy-preview-1517--regen-storybook.netlify.app/?path=/story/modal--tx-successful-modal now includes an example of a description with a link that opens a new tab
<img width="642" alt="Screen Shot 2022-10-25 at 11 36 38 AM" src="https://user-images.githubusercontent.com/9438198/197843525-be133bbd-8ca1-4e90-a933-7fe350a7ccd9.png">

This is the only element I see as necessary to creating this success modal (mock):

<img width="656" alt="Screen Shot 2022-10-25 at 11 38 19 AM" src="https://user-images.githubusercontent.com/9438198/197843873-a2340be8-2939-47e8-88b1-1ed0bf5756cb.png">

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...
